src/parameters/parameters_ode_solver.cpp

```diff
@@ -22,6 +22,25 @@ void ODESolverParam::declare_parameters (dealii::ParameterHandler &prm)
                           dealii::Patterns::Double(0,dealii::Patterns::Double::max_double_value),
                           "Outputs the solution at time intervals of dt in .vtk file");
 
+        prm.declare_entry("output_solution_at_fixed_times", "false",
+                          dealii::Patterns::Bool(),
+                          "Output solution at fixed times. False by default.");
+
+        prm.declare_entry("output_solution_fixed_times_string", " ",
+                          dealii::Patterns::FileName(dealii::Patterns::FileName::FileType::input),
```

Comment:
See `parameter_min_values` in `parameters_reduced_order` to possibly replace this with a list rather than a string. (Note that I haven't used this functionality myself, just that I know it exists!)

Reply:
He seems to be using it expecting that 2 values are always passed, I think we can leave it as this implementation works and it doesn't affect how things are written in the `.prm` file

Reply:
Sounds good. The way you're currently doing it is fine, just thought to point that out in case it made the task easier!